### PR TITLE
Expose normalised data into async response

### PR DESCRIPTION
**Pull request by CAPSLOCKUSER**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,6 +241,17 @@ const mapDispatchToProps = (dispatch, props) => {
 };
 ```
 
+The result of the promise returned by `mutateAsync` will be the following object:
+
+| Name | Type | Description |
+|:----|:-----|:-----|
+| status | number | HTTP status code
+| body | object | Response body auto-parsed by SuperAgent
```
Comment (ryanashcraft) [inline README.md]: change to "Parsed response body." prefer not to mention superagent soon (see #23) will support other libs that will provide the same interface

File: README.md [hunk 2]
```diff
@@ -240,6 +241,17 @@ const mapDispatchToProps = (dispatch, props) => {
...
+| text | string | Unparsed response body string
+| duration | number | Elapsed time between the action dispatched and resolved
+| transformed | any | Normalized data by the `transform` function
+| entities | object | The new entity collection
```
Comment (ryanashcraft) [inline README.md]: change to "The new, updated entities that have been affected by the query."

File: README.md [hunk 3]
```diff
@@ -240,6 +241,17 @@ const mapDispatchToProps = (dispatch, props) => {
...
 
+The result of the promise returned by `mutateAsync` will be the following object:
+
+| Name | Type | Description |
+|:----|:-----|:-----|
+| status | number | HTTP status code
+| body | object | Response body auto-parsed by SuperAgent
+| text | string | Unparsed response body string
+| duration | number | Elapsed time between the action dispatched and resolved
+| transformed | any | Normalized data by the `transform` function
```
Comment (ryanashcraft) [inline README.md]: Change to "Result from the `transform` function. Will be identical to `body` if `transform` is unprovided in the query config."

File: README.md [hunk 4]
```diff
@@ -240,6 +241,17 @@ const mapDispatchToProps = (dispatch, props) => {
...
 ```
 
+The result of the promise returned by `mutateAsync` will be the following object:
+
+| Name | Type | Description |
+|:----|:-----|:-----|
+| status | number | HTTP status code
+| body | object | Response body auto-parsed by SuperAgent
+| text | string | Unparsed response body string
+| duration | number | Elapsed time between the action dispatched and resolved
```
Comment (ryanashcraft) [inline README.md]: I'd prefer to keep it high level. Something like "The total duration from the start of the query to receiving the full response."

File: README.md (hunk 1 above)
Comment (ryanashcraft) [inline README.md]: also change type to "object" to "object or null"

**Comment by CAPSLOCKUSER** [comment]
Done!

**Comment by ryanashcraft** [comment]
Thanks @CAPSLOCKUSER

**Pull request merged**